src/engine/ratecontrol.h

```diff
@@ -34,6 +34,31 @@ class RateControl : public EngineControl {
     RateControl(QString group, UserSettingsPointer pConfig);
     virtual ~RateControl();
 
+    // Enumerations which hold the state of the pitchbend buttons.
+    // These enumerations can be used like a bitmask.
+    enum RATERAMP_DIRECTION {
+        RATERAMP_NONE = 0,  // No buttons are held down
+        RATERAMP_DOWN = 1,  // Down button is being held
+        RATERAMP_UP = 2,    // Up button is being held
+        RATERAMP_BOTH = 3   // Both buttons are being held down
+    };
+
+    enum class RampMode {
+        Stepping = 0, // pitch takes a temporary step up/down a certain amount
```

Comment:
Enum value names should be capital case

Reply:
Are you sure? I think we are using camel-case for enumerated values, at least for enum classes. Google allows both constant and macro style. IMHO the camel case constant style is more pleasant for reading.

Google also suggests to use a "k" prefix like for constants, but this is IMHO ridiculous for enum class values: RampMode::Stepping vs. RampMode::kStepping

Reply:
RampMode::STEPPING shows how the capitalized macro style breaks consistency, even more for literals with multiple terms.

Reply:
I've updated the coding guidelines, because an advice for enumerated values was missing. Feel free to discuss or change it I got this wrong.

https://mixxx.org/wiki/doku.php/coding_guidelines#forward_declared_strongly_typed_enums

There are still a lot of capitalized enum class enumerators in the code that need to be migrated at some time. I also used this style in the past.

Reply:
@daschuer Removed the "Bad" example. Using C-style enums indeed is still an option. The section should just show a typical example for the new C++-style enums that are the preferred way.